### ISSUE TITLE
hello,something about finetune myself dataset about head detect?

### ISSUE THREAD

**Issue by zzy0724** [YonghaoHe/A-Light-and-Fast-Face-Detector-for-Edge-Devices#21]
hello,Thanks for sharing。
Can you share the specific training process?Because I want to fine-tune my dataset

**Comment by YonghaoHe** [comment]
@zzy0724 The training instructions of LFFD is specified in face_detection subdir. You may learn necessary steps there. In general, fine-tuning may start with learning rate 0.01, training from scratch with learning rate 0.1.

**Comment by zzy0724** [comment]
Thank you for your reply!
due to brainwash dataset  is not available online,it may be deleted by author.Could you send me it ?
my email is  1057610223@qq.com
Thank you again!

**Comment by YonghaoHe** [comment]
@zzy0724 we will upload packed pkl brainwash dataset in recent days.

**Comment by zzy0724** [comment]
Thank you!
Online waiting for...

**Comment by YonghaoHe** [comment]
@zzy0724 The brainwash dataset is uploaded now.

**Comment by zzy0724** [comment]
Thank you very much!

**Comment by zzy0724** [comment]
Hello, I have a problem with the preparation of the human head detection data set,Can you help me out?
queston:i want to collect my own dataset,I use open source tool of  labellmg to label.but l don't know how to convert it to the file of pkl and the specific content of data_list_train.txt.
Looking forward to your reply!!!
